references/firestore-facade/sf-firestore-facade-populate-v1/sharedflowbundle/policies/SC-PopulateCache.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2023 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ServiceCallout continueOnError="true" name="SC-PopulateCache">
+    <Request>
+        <Set>
+            <Headers>
+                <Header name="Accept">application/json</Header>
+            </Headers>
+            <QueryParams>
+                <QueryParam name="documentId">{flow.pathSuffix}</QueryParam>
+            </QueryParams>
+            <Verb>POST</Verb>
+            <Payload contentType="application/json">{"fields": {"data": {"stringValue": "{flow.jsonContentAsString}"}}}</Payload>
+        </Set>
+        <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
+    </Request>
+    <Response>firestoreCacheResponse</Response>
+    <HTTPTargetConnection>
+        <Authentication>
+            <GoogleAccessToken>
+                <Scopes>
+                    <Scope>https://www.googleapis.com/auth/cloud-platform</Scope>
+                </Scopes>
+            </GoogleAccessToken>
+        </Authentication>
+        <URL>https://firestore.googleapis.com/v1beta1/projects/{organization.name}/databases/(default)/documents/{flow.basePath}?documentId={flow.pathSuffix}</URL>
```

Comment:
Does `?documentId` need to be included here if it's being set by `<Set>` above?

Reply:
done